README.md

```diff
@@ -26,13 +26,14 @@ Before you can start the initialization of Affirm SDK, you must first set the Af
 ```java
 Affirm.initialize(new Affirm.Configuration.Builder("public key")
         .setEnvironment(Affirm.Environment.SANDBOX)
+        .setCountryCode("USA")  // Default USA
```

Comment:
Would it make more sense to use `Locale.US.country` here instead of hardcoded strings?